src/shared/features/AppletSettings/DataRetentionSetting/DataRetention.tsx

```diff
@@ -77,6 +78,11 @@ export const DataRetention = () => {
     }
   };
 
+  const handleDontSave = () => {
+    reset(defaultValues);
```

Comment:
I'm curious how reset to defeault values helps to close the popup?

Reply:
usePrompt(isDirty && !isSubmitted);

because if isDirty